### ISSUE TITLE
LLVM errors

### ISSUE THREAD

**Issue by mosheduminer** [JuliaLang/julia#36902]
I was playing with the `CombinedParsers.jl` library, when I got an LLVM error. Upon further attempts with related code, I got other LLVM errors. The following is the code for the first error:
```julia
julia> using CombinedParsers

julia> nlb = NegativeLookbehind("(:")
re"(?<!\(\:)"

julia> unicode = parser(CharIn(isvalid))
re"[#$(isvalid)]"

julia> parse(unicode * nlb, "test(:")
```

Running the last line (this was in the REPL) caused the following output:
```
Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x28b1ea8 -- _ZN4llvm11PointerType3getEPNS_4TypeEj at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
in expression starting at REPL[7]:1
_ZN4llvm11PointerType3getEPNS_4TypeEj at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner18visitAddrSpaceCastERNS_17AddrSpaceCastInstE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner3runEv at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZL31combineInstructionsOverFunctionRN4llvm8FunctionERNS_19InstCombineWorklistEPNS_9AAResultsERNS_15AssumptionCacheERNS_17TargetLibraryInfoERNS_13DominatorTreeERNS_25OptimizationRemarkEmitterEPNS_18BlockFrequencyInfoEPNS_18ProfileSummaryInfoEbPNS_8LoopInfoE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm24InstructionCombiningPass13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager11runOnModuleERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm6legacy15PassManagerImpl3runERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
operator() at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:520
addModule at /cygdrive/d/buildbot/worker/package_win64/build/usr/include/llvm/ExecutionEngine/Orc\IRCompileLayer.h:93 [inlined]
addModule at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:648
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:893
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:955
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:977 [inlined]
_jl_compile_codeinst at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:126
jl_generate_fptr at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:302
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1964
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1931 [inlined]
_jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2224 [inlined]
jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2238
iterate at .\iterators.jl:1064
unknown function (ip: 000000002369BDF0)
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:628
sSequence at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1598 [inlined]
* at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1795
unknown function (ip: 000000002369BCBD)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_call at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:117
eval_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:206
eval_stmt_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:157 [inlined]
eval_body at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:548
jl_interpret_toplevel_thunk at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:660
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:840
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:790
jl_toplevel_eval at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:849 [inlined]
jl_toplevel_eval_in at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:883
eval at .\boot.jl:331
eval_user_input at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:134
repl_backend_loop at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:195
start_repl_backend at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:180
#run_repl#37 at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:292
run_repl at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:288
#806 at .\client.jl:399
jfptr_YY.806_58616.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:655
jl_f__apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:669 [inlined]
jl_f__apply_latest at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:705
#invokelatest#1 at .\essentials.jl:710 [inlined]
invokelatest at .\essentials.jl:709 [inlined]
run_main_repl at .\client.jl:383
exec_options at .\client.jl:313
_start at .\client.jl:506
jfptr__start_54312.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
unknown function (ip: 00000000004017E1)
unknown function (ip: 0000000000401BD6)
unknown function (ip: 00000000004013DE)
unknown function (ip: 000000000040151A)
BaseThreadInitThunk at C:\WINDOWS\System32\KERNEL32.DLL (unknown line)
RtlUserThreadStart at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
Allocations: 25850357 (Pool: 25839969; Big: 10388); GC: 27
LLVM ERROR: starting new .cfi frame before finishing the previous one
```

The code for the second error is as follows:

```julia
julia> using CombinedParsers

julia> unicode = parser(CharIn(isvalid))
re"[#$(isvalid)]"

julia> nlb = NegativeLookbehind("help")
re"(?<!help)"

julia> parse(unicode * nlb, "help")
```

Running the last line (in the REPL) caused the following output:
```
Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: UNKNOWN at 0x2771ea8 -- _ZN4llvm11PointerType3getEPNS_4TypeEj at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
in expression starting at REPL[7]:1
_ZN4llvm11PointerType3getEPNS_4TypeEj at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner18visitAddrSpaceCastERNS_17AddrSpaceCastInstE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner3runEv at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZL31combineInstructionsOverFunctionRN4llvm8FunctionERNS_19InstCombineWorklistEPNS_9AAResultsERNS_15AssumptionCacheERNS_17TargetLibraryInfoERNS_13DominatorTreeERNS_25OptimizationRemarkEmitterEPNS_18BlockFrequencyInfoEPNS_18ProfileSummaryInfoEbPNS_8LoopInfoE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm24InstructionCombiningPass13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager11runOnModuleERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm6legacy15PassManagerImpl3runERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
operator() at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:520
addModule at /cygdrive/d/buildbot/worker/package_win64/build/usr/include/llvm/ExecutionEngine/Orc\IRCompileLayer.h:93 [inlined]
addModule at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:648
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:893
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:955
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:977 [inlined]
_jl_compile_codeinst at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:126
jl_generate_fptr at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:302
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1964
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1931 [inlined]
_jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2224 [inlined]
jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2238
iterate at .\iterators.jl:1064
unknown function (ip: 0000000023596EF0)
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:628
sSequence at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1598 [inlined]
* at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1795
unknown function (ip: 0000000023596DBD)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_call at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:117
eval_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:206
eval_stmt_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:157 [inlined]
eval_body at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:548
jl_interpret_toplevel_thunk at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:660
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:840
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:790
jl_toplevel_eval at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:849 [inlined]
jl_toplevel_eval_in at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:883
eval at .\boot.jl:331
eval_user_input at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:134
repl_backend_loop at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:195
start_repl_backend at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:180
#run_repl#37 at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:292
run_repl at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:288
#806 at .\client.jl:399
jfptr_YY.806_58616.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:655
jl_f__apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:669 [inlined]
jl_f__apply_latest at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:705
#invokelatest#1 at .\essentials.jl:710 [inlined]
invokelatest at .\essentials.jl:709 [inlined]
run_main_repl at .\client.jl:383
exec_options at .\client.jl:313
_start at .\client.jl:506
jfptr__start_54312.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
unknown function (ip: 00000000004017E1)
unknown function (ip: 0000000000401BD6)
unknown function (ip: 00000000004013DE)
unknown function (ip: 000000000040151A)
BaseThreadInitThunk at C:\WINDOWS\System32\KERNEL32.DLL (unknown line)
RtlUserThreadStart at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
Allocations: 26774371 (Pool: 26763595; Big: 10776); GC: 23

Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x30258aa -- _ZN4llvm12InstCombiner14visitStoreInstERNS_9StoreInstE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
in expression starting at REPL[7]:1
_ZN4llvm12InstCombiner14visitStoreInstERNS_9StoreInstE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner3runEv at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZL31combineInstructionsOverFunctionRN4llvm8FunctionERNS_19InstCombineWorklistEPNS_9AAResultsERNS_15AssumptionCacheERNS_17TargetLibraryInfoERNS_13DominatorTreeERNS_25OptimizationRemarkEmitterEPNS_18BlockFrequencyInfoEPNS_18ProfileSummaryInfoEbPNS_8LoopInfoE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm24InstructionCombiningPass13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager11runOnModuleERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm6legacy15PassManagerImpl3runERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
operator() at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:520
addModule at /cygdrive/d/buildbot/worker/package_win64/build/usr/include/llvm/ExecutionEngine/Orc\IRCompileLayer.h:93 [inlined]
addModule at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:648
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:893
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:955
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:940
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:940
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:977 [inlined]
_jl_compile_codeinst at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:126
jl_generate_fptr at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:302
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1964
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1931 [inlined]
_jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2224 [inlined]
jl_apply_generic at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2398
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
jl_atexit_hook at /cygdrive/d/buildbot/worker/package_win64/build/src\init.c:230
jl_exit at /cygdrive/d/buildbot/worker/package_win64/build/src\jl_uv.c:624
jl_exception_handler at /cygdrive/d/buildbot/worker/package_win64/build/src\signals-win.c:308
__julia_personality at /cygdrive/d/buildbot/worker/package_win64/build/src/support\win32_ucontext.c:28
_chkstk at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
RtlRaiseException at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
KiUserExceptionDispatcher at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
_ZN4llvm11PointerType3getEPNS_4TypeEj at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner18visitAddrSpaceCastERNS_17AddrSpaceCastInstE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm12InstCombiner3runEv at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZL31combineInstructionsOverFunctionRN4llvm8FunctionERNS_19InstCombineWorklistEPNS_9AAResultsERNS_15AssumptionCacheERNS_17TargetLibraryInfoERNS_13DominatorTreeERNS_25OptimizationRemarkEmitterEPNS_18BlockFrequencyInfoEPNS_18ProfileSummaryInfoEbPNS_8LoopInfoE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm24InstructionCombiningPass13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager13runOnFunctionERNS_8FunctionE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm13FPPassManager11runOnModuleERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
_ZN4llvm6legacy15PassManagerImpl3runERNS_6ModuleE at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\bin\LLVM.dll (unknown line)
operator() at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:520
addModule at /cygdrive/d/buildbot/worker/package_win64/build/usr/include/llvm/ExecutionEngine/Orc\IRCompileLayer.h:93 [inlined]
addModule at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:648
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:893
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:955
jl_add_to_ee at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:977 [inlined]
_jl_compile_codeinst at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:126
jl_generate_fptr at /cygdrive/d/buildbot/worker/package_win64/build/src\jitlayers.cpp:302
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1964
jl_compile_method_internal at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:1931 [inlined]
_jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2224 [inlined]
jl_invoke at /cygdrive/d/buildbot/worker/package_win64/build/src\gf.c:2238
iterate at .\iterators.jl:1064
unknown function (ip: 0000000023596EF0)
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:628
sSequence at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1598 [inlined]
* at C:\Users\moshe\.julia\packages\CombinedParsers\1ECpL\src\CombinedParsers.jl:1795
unknown function (ip: 0000000023596DBD)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_call at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:117
eval_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:206
eval_stmt_value at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:157 [inlined]
eval_body at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:548
jl_interpret_toplevel_thunk at /cygdrive/d/buildbot/worker/package_win64/build/src\interpreter.c:660
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:840
jl_toplevel_eval_flex at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:790
jl_toplevel_eval at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:849 [inlined]
jl_toplevel_eval_in at /cygdrive/d/buildbot/worker/package_win64/build/src\toplevel.c:883
eval at .\boot.jl:331
eval_user_input at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:134
repl_backend_loop at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:195
start_repl_backend at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:180
#run_repl#37 at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:292
run_repl at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\REPL\src\REPL.jl:288
#806 at .\client.jl:399
jfptr_YY.806_58616.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
jl_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\julia.h:1690 [inlined]
do_apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:655
jl_f__apply at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:669 [inlined]
jl_f__apply_latest at /cygdrive/d/buildbot/worker/package_win64/build/src\builtins.c:705
#invokelatest#1 at .\essentials.jl:710 [inlined]
invokelatest at .\essentials.jl:709 [inlined]
run_main_repl at .\client.jl:383
exec_options at .\client.jl:313
_start at .\client.jl:506
jfptr__start_54312.clone_1 at C:\Users\moshe\AppData\Local\Programs\Julia 1.5.0\lib\julia\sys.dll (unknown line)
unknown function (ip: 00000000004017E1)
unknown function (ip: 0000000000401BD6)
unknown function (ip: 00000000004013DE)
unknown function (ip: 000000000040151A)
BaseThreadInitThunk at C:\WINDOWS\System32\KERNEL32.DLL (unknown line)
RtlUserThreadStart at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
Allocations: 27632847 (Pool: 27621831; Big: 11016); GC: 24
```

I am sure that the implementation of CombinedParsers is relevant here, but I have no idea how it works.

`versioninfo()` output:
```julia
julia> versioninfo()
Julia Version 1.5.0
Commit 96786e22cc (2020-08-01 23:44 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
```

**Comment by gkappler** [comment]
I can reproduce this error on linux with Julia 1.5.0.
The example is working as expected in 1.4.1.

**Comment by mosheduminer** [comment]
I have prepared rr traces. Interestingly, on ubuntu linux I get segmentation fault errors, rather than the errors I posted above.
rr trace for the first code block:

https://s3.amazonaws.com/julialang-dumps/reports/2020-08-04T13-22-00-mosheduminer.tar.zst

rr trace for the second code block:

https://s3.amazonaws.com/julialang-dumps/reports/2020-08-04T13-24-08-mosheduminer.tar.zst

**Comment by Keno** [comment]
Hmm, there seems to be something wrong with those traces - they appear to be corrupted. Did something go wrong during recording?

**Comment by mosheduminer** [comment]
> Did something go wrong during recording?

Not that I am aware of... I have uploaded another trace [here](https://s3.amazonaws.com/julialang-dumps/reports/2020-08-09T01-07-59-mosheduminer.tar.zst), for the first code block. Is this one okay?

**Comment by Keno** [comment]
Your traces keep ending up being exactly 76 bytes. Are you perhaps on an unsupported platform - maybe the code that is supposed to error out there broke?

**Comment by mosheduminer** [comment]
I did the traces on an ubuntu distro (zorin 15, based on 18.04). I don't know why they are corrupted... 🤷

**Comment by Keno** [comment]
@SimonDanisch complained that some of the fatal errors weren't sufficiently noisy (https://github.com/JuliaLang/BugReporting.jl/issues/24), perhaps you had the same issue?

**Comment by mosheduminer** [comment]
I don't know... And as I mentioned, I got segfaults on linux, instead of the LLVM errors I got on windows.

**Comment by vtjnash** [comment]
Fixed by #39801 (confirmed by backporting that commit, since CombinedParsers.jl doesn't currently work on v1.6)
